tests/onnxruntime/test_modeling.py

```diff
@@ -2138,11 +2138,18 @@ def test_compare_to_transformers(self, model_arch):
 
         for input_type in ["pt", "np"]:
             tokens = tokenizer(text, return_tensors=input_type)
+            # Test default behavior (return_dict=False)
             onnx_outputs = onnx_model(**tokens)
-
+            self.assertIsInstance(onnx_outputs, BaseModelOutput)
             self.assertIn("last_hidden_state", onnx_outputs)
             self.assertIsInstance(onnx_outputs.last_hidden_state, self.TENSOR_ALIAS_TO_TYPE[input_type])
 
+            # Test return_dict=True
+            onnx_outputs_dict = onnx_model(**tokens, return_dict=True)
+            self.assertIsInstance(onnx_outputs_dict, dict)
+            self.assertIn("last_hidden_state", onnx_outputs_dict)
+            self.assertIsInstance(onnx_outputs_dict["last_hidden_state"], self.TENSOR_ALIAS_TO_TYPE[input_type])
```

Comment:
nice ! we can generalise this to other classes after the patch, for now this is enough to unblock sentence transformers